types/mathjs/index.d.ts

```diff
@@ -1,2252 +1,4546 @@
-// Type definitions for mathjs 3.21
+// Type definitions for mathjs 4.4.2
```

Comment:
Because patch versions (`major.minor.patch`) shouldn't introduce type changes, DefinitelyTyped doesn't care about the patch version.

The linter wants this to be `// Type definitions for mathjs 4.4`